lib/yggdrasil/player.ex

```diff
@@ -12,9 +13,16 @@ defmodule Yggdrasil.Player do
     Supervisor.add_player user_id, [owner_pid, push_msg]
   end
 
-  def run_cmd(user_id, cmd = %Message{type: :command}) do
-    player_pid = Registry.get_player user_id
-    GenServer.cast(player_pid, {:run_cmd, cmd})
+  def run_cmd(user_id, cmd) do
+    # horrible terrible temporary implementation
+    # room would eventually do all of this
+    ctxt = %Yggdrasil.Room.Context{ player: user_id }
+    ctxt = Command.execute cmd, ctxt
+
+    Enum.each ctxt.actions,
+      fn {:notify, player, message} ->
+        notify(player, message)
+      end
```

Comment:
I know this looks terrible but the idea is to separate who performs the mutations which this accomplishes. I think once a game world is built and rooms are handling commands we'll move this all out.